odoo/addons/base/models/ir_actions_report.py

```diff
@@ -395,6 +395,7 @@ def _run_wkhtmltopdf(
             bodies,
             header=None,
             footer=None,
+            report_ref=False,
```

Comment:
I would not make it optional with a default value of False.
I think that it should be mandatory as if we don't have one it will raise a traceback in `_get_report()`
We should also update the docstring to add this param :smile:

Reply:
There is a lot of calls to `_run_wkhtmltopdf` in Odoo, if you change the signature, you need to correct them all.